cmd/broker/retry/kodata/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
There should be more stuff in kodata? And they should be symlinks. See my PR https://github.com/google/knative-gcp/commit/acedadbe224d707f0a3b3fcd34d9d9643e734c94